app/database/models/server/user.ts

```diff
@@ -141,4 +146,35 @@ export default class UserModel extends Model {
             u.status = status;
         });
     }
+
+    get mentionKeys() {
```

Comment:
this is great.. we do have  a function called `getUserMentionKeys` under `app/utils/user` and I wonder if we should make use of this in that function as well